### ISSUE TITLE
[Feature Request] An option to disable the auto closing mechanism of Deceive

### ISSUE THREAD

**Issue by XamarinDeveloper** [molenzwiebel/Deceive#97]
I know the auto closing mechanism is needed and why, but in my system sometimes when I start the game until the game fully opens the Deceive app closes and it's really frustrating , I'd like an option to disable the auto closing so it won't happen again. Thanks.

P.S: I wanted to add it myself but for some reason my VS won't compile the project so, I'm asking here :)

**Comment by aPinat** [comment]
I don't quite understand what you mean by the auto-closing. Deceive does not just close by itself.
Deceive only exits when the LeagueClient itself exits, or when Deceive encounters an error and crashes.

**Comment by XamarinDeveloper** [comment]
Yeah, sorry for not explaining that part, the "when the LeagueClient exits" thingy I mean, 'cause the crashes have a MessageBox pop up but I don't get any message, just the app stops, so I guessed it's because of that auto-closing hence this request

P.S: I'm using the app for VALORANT idk if that has to do with anything 🤔

**Comment by aPinat** [comment]
So the thing is, when Deceive crashes or closes, the game you're launching with it also ceases to function properly to some extend because of how social features are deeply tied into some games. In VALORANT this would be the friends list being empty, the party system not functioning and in match chat not working. This limbo state is mitigated by having Deceive close the game it launched, since not closing it would potentially lead to more problems. 
I would rather fix the issue that makes Deceive crash in the first place, since that shouldn't just happen out of the blue. Having your proposed option would make the current context menu, which is already a bit confusing and cluttered, even worse to understand, which in my opinion has no real use for general users anyways.

This is how I see it, in the end it's for repo owner `@molenzwiebel` to decide, so...

You should be able to compile the project out of the box though since Visual Studio doesn't need anything special for this, especially if you are already using the Xamarin framework (or is that unrelated to your name?).

**Comment by XamarinDeveloper** [comment]
Yeah I get that it's not the best solution, and yes finding the reason of the crash is the best way :)

Idk, the dependencies won't load and when I try to build the project I get this:

`========== Build: 0 succeeded, 0 failed, 1 up-to-date, 0 skipped ==========`

and when I try to run I get the error that the exec file does not exist.

**Comment by aPinat** [comment]
That one line doesn't tell me anything, looks like you didn't even build and only pulled deps or something...
